crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -445,18 +446,33 @@ impl TryFrom<&LogletConfig> for LogletConfiguration {
     }
 }
 
+#[derive(
+    Default, Eq, PartialEq, PartialOrd, Ord, Clone, Copy, derive_more::From, derive_more::Into,
+)]
+struct TrialCounter(u8);
```

Comment:
Minor nit but could we may be use `Attempt(s)Counter` to just to align terminology with `RetryPolicy`?